jetbrains-core/src/software/aws/toolkits/jetbrains/core/executables/ExecutableManager.kt

```diff
@@ -187,9 +220,10 @@ class DefaultExecutableManager : PersistentStateComponent<ExecutableStateList>,
 }
 
 sealed class ExecutableInstance {
+    abstract val version: String?
```

Comment:
The ability to get version on an UnresolvedExecutable. For example, here:
```
        fun getVersionString(): String = ExecutableManager.getInstance().getExecutableIfPresent<SamExecutable>().version ?: "UNKNOWN"
```
it's useful because an UnresolvedExecutable still has a version (it's null/unkown) and in every case we access version we don't have to check if it's unresolved or a `ExecutableWithPath`.

Reply:
Fair enough